cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1459,7 +1459,7 @@ def test_json_responses(self):
         self.validate_course_xblock_info(json_response, course_outline=True)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 5, 5),
+        (ModuleStoreEnum.Type.split, 4, 4),
```

Comment:
I am guessing, but perhaps it has to do with the change from Scope.content to Scope.settings -- it could be that an additional ModuleStore call is not required to access Scope.content information with the field relocation.